tools.md

```diff
@@ -14,16 +14,52 @@ sidenav:
 ## USWDS Component Library in Figma
 
 ## Figma tokens plug-in
+#### What is Figma Tokens? ####
+The Figma Tokens plugin allows you to integrate tokens into your designs in Figma. There are three approaches to begin using the Tokens plugin:
+  1. Create your tokens from scratch, 
+  2. Import your existing styles into the Tokens plugin, 
+  3. Or create tokens using the built-in JSON editor. 
 
-- item
-- item
-- item
+#### Importing tokens ####
+- If you're working with an existing design system and you've already defined font styles, colors, spacing, etc. in Figma, then use the import feature to generate tokens based on those existing styles. 
+- This can be the easiest way to start using tokens because the plugin will auto-generate the tokens based on what you have in your system, instead of starting from nothing. 
 
-## Other Figma plug-ins
+![import button on tokens plugin](/assets/img/import-styles.jpeg)
+
+- After importing existing styles to create tokens, you can adjust the token names with your developer and come up with the naming conventions that make the most senese for you.
+
+- If your developer is already using tokens, you can use the JSON editor in the tokens plugin to import and begin designing from what they already have. 
+
+![json editor in plugin](/assets/img/json-view.png)
+
+#### Some Considerations ####
```

Comment:
This heading should be nested in the Importing Tokens section. So when changing that one to an H2, this one should be an H3.